systemtest/src/test/java/io/strimzi/systemtest/kafka/listeners/ListenersST.java

```diff
@@ -668,6 +668,92 @@ void testLoadBalancerTls(ExtensionContext extensionContext) {
         );
     }
 
+    @ParallelNamespaceTest
+    @Tag(SANITY)
+    @Tag(ACCEPTANCE)
```

Comment:
I think we do not want it in the ACCEPTANCE test suite since it would make it too long. I think this should say `@Tag(REGRESSION)` for both of the tests, or @im-konge

Reply:
I agree with @scholzj , `SANITY` profile contains tests for each component (just a sanity check that everything is working). When you take a look on a `testNodePort` and other, we are running it just for regression. So I would go this way here too.

You don't need to add the `REGRESSION` tag, it will be taken from the whole class.

Reply:
finally I have to keep SANITY and remove ACCEPTANCE, right?

Reply:
Remove the `SANITY` too please, as these don't need to be in this group of sanity tests